cypress/Pages/Login/index.js

```diff
@@ -0,0 +1,46 @@
+import Page from "../page";
+import { Selectors } from "../../../src/components/Login/selectors";
+
+class LoginPage extends Page {
+  get emailInput() {
+    return cy.get(Selectors.emailInput);
+  }
+
+  get passwordInput() {
+    return cy.get(Selectors.passwordInput);
+  }
+
+  get loginButton() {
+    return cy.get(Selectors.loginButton);
+  }
+
+  get invalidCredentialsAlert() {
+    return cy.get(Selectors.invalidCredentialsAlert);
+  }
+
+  get invalidInputAlert() {
+    return cy.get(Selectors.invalidInputAlert);
+  }
+
+  get networkErrorAlert() {
+    return cy.get(Selectors.networkErrorAlert);
+  }
+
+  open() {
+    super.open("login");
+  }
+
+  login(email, password) {
```

Comment:
Better to make the function general like this rather than have specific login functions